xbmc/music/MusicDatabase.cpp

```diff
@@ -10874,9 +11036,12 @@ void CMusicDatabase::SetPropertiesFromAlbum(CFileItem& item, const CAlbum& album
   if (album.iVotes > 0)
     item.SetProperty("album_votes", album.iVotes);
 
+  item.SetProperty("album_releasesdate", album.m_strDateOfRelease); // only a year ?
   item.SetProperty("album_isboxset", album.bBoxedSet);
   item.SetProperty("album_totaldiscs", album.iTotalDiscs);
   item.SetProperty("album_releasetype", CAlbum::ReleaseTypeToString(album.releaseType));
+  item.SetProperty("album_albumreleasedate", album.strAlbumReleaseDate); //could be full date
+  item.SetProperty("album_originaldate", album.strOrigReleaseDate);
```

Comment:
The album_XXX properties are put in map here to give the extra album data for a song item. 
A song item does not need original and release date values as album properies, it already has these values itself.  Remember these dates are held as  album values (denormalised) for convenience of lookup, and  should be the same as for the songs.

Reply:
OK